lc-hax/Scripts/Helpers/Color.cs

```diff
@@ -0,0 +1,170 @@
+        internal static Color BurlyWood { get; } = new(0.8705882f, 0.7215686f, 0.5294118f, 1f);
+        internal static Color DarkOliveGreen { get; } = new(0.3333333f, 0.4196078f, 0.1843137f, 1f);
+        internal static Color CornflowerBlue { get; } = new(0.3921569f, 0.5843138f, 0.9294118f, 1f);
+        internal static Color DarkOrchid { get; } = new(0.6f, 0.1960784f, 0.8f, 1f);
+        internal static Color Khaki { get; } = new(0.9411765f, 0.9019608f, 0.5490196f, 1f);
+        internal static Color Ivory { get; } = new(1f, 1f, 0.9411765f, 1f);
+        internal static Color DarkOrange { get; } = new(1f, 0.5490196f, 0f, 1f);
+        internal static Color Indigo { get; } = new(0.2941177f, 0f, 0.509804f, 1f);
+        internal static Color IndianRed { get; } = new(0.8039216f, 0.3607843f, 0.3607843f, 1f);
+        internal static Color HotPink { get; } = new(1f, 0.4117647f, 0.7058824f, 1f);
+        internal static Color Honeydew { get; } = new(0.9411765f, 1f, 0.9411765f, 1f);
+        internal static Color GreenYellow { get; } = new(0.6784314f, 1f, 0.1843137f, 1f);
+        internal static Color Green { get; } = new(0f, 0.5019608f, 0f, 1f);
+        internal static Color Gray { get; } = new(0.5019608f, 0.5019608f, 0.5019608f, 1f);
+        internal static Color Goldenrod { get; } = new(0.854902f, 0.6470588f, 0.1254902f, 1f);
+        internal static Color GhostWhite { get; } = new(0.972549f, 0.972549f, 1f, 1f);
+        internal static Color Gainsboro { get; } = new(0.8627451f, 0.8627451f, 0.8627451f, 1f);
+        internal static Color Fuchsia { get; } = new(1f, 0f, 1f, 1f);
+        internal static Color Gold { get; } = new(1f, 0.8431373f, 0f, 1f);
+        internal static Color FloralWhite { get; } = new(1f, 0.9803922f, 0.9411765f, 1f);
+        internal static Color DarkRed { get; } = new(0.5450981f, 0f, 0f, 1f);
+        internal static Color DarkSalmon { get; } = new(0.9137255f, 0.5882353f, 0.4784314f, 1f);
+        internal static Color DarkSeaGreen { get; } = new(0.5607843f, 0.7372549f, 0.5450981f, 1f);
+        internal static Color ForestGreen { get; } = new(0.1333333f, 0.5450981f, 0.1333333f, 1f);
+        internal static Color DarkSlateGray { get; } = new(0.1843137f, 0.3098039f, 0.3098039f, 1f);
+        internal static Color DarkTurquoise { get; } = new(0f, 0.8078431f, 0.8196079f, 1f);
+        internal static Color DarkSlateBlue { get; } = new(0.282353f, 0.2392157f, 0.5450981f, 1f);
+        internal static Color DeepPink { get; } = new(1f, 0.07843138f, 0.5764706f, 1f);
+        internal static Color DeepSkyBlue { get; } = new(0f, 0.7490196f, 1f, 1f);
+        internal static Color DimGray { get; } = new(0.4117647f, 0.4117647f, 0.4117647f, 1f);
+        internal static Color DodgerBlue { get; } = new(0.1176471f, 0.5647059f, 1f, 1f);
+        internal static Color Firebrick { get; } = new(0.6980392f, 0.1333333f, 0.1333333f, 1f);
+        internal static Color DarkViolet { get; } = new(0.5803922f, 0f, 0.827451f, 1f);
+    }
+}
```

Comment:
Where did you steal this from?

Reply:
That's from AstroClient, my old VRChat client i did awhile ago.

Reply:
You handwrote all of these colours?

Reply:
more like i made a utility to convert the entire System.Colors to unity and then i converted the results into a struct

Reply:
ESP identification looks way better than the unity stock color options.
i also include a way to convert Hex to Unity color directly, but is better if you do it  into a property or field and go from there.

Reply:
<img width="1140" alt="image" src="https://github.com/winstxnhdw/lc-hax/assets/92211398/a5bc4fcf-55f3-42fb-b4db-e7a2d859aac3">